sql-statements/sql-statement-commit.md

```diff
@@ -17,6 +17,10 @@ In the absence of a `BEGIN` or `START TRANSACTION` statement, the default behavi
 
 ![CommitStmt](/media/sqlgram/CommitStmt.png)
 
+**CompletionTypeWithinTransaction:**
+
+![CompletionTypeWithinTransaction](/media/sqlgram/CompletionTypeWithinTransaction.png)
+
 ## Examples
 
 ```sql
```

Comment:
For line 42-44, see https://github.com/pingcap/docs-cn/pull/3301/files?file-filters%5B%5D=.md#r441470759

Reply:
This sentence has been added in [line44](https://github.com/pingcap/docs/pull/2922/files#diff-4bc4de2792f4bfc92c0f73ec06393007R44) .

Reply:
And line 42-43 need to be deleted? @zimulala

Reply:
Problem solved. No need to delete them. Can merge now.

Reply:
No need. Just like the previous sentence, PR especially changed it at the back.